docs/articles/rag-application-communication-system.md

```diff
@@ -0,0 +1,164 @@
+
+It sounds like a familiar pattern. RAG emerged with a similar promise of simplification shock: taking whatever content there is, outputting whatever content the user wants. This is not only the obvious objective of LLM on the output side but also embedding models on the input side. Embeddings are natively multilingual, theoretically resilient to synonyms, variations of language standards and document artifacts. 
+
+The radical promise frequently failed on two fronts: understanding users, understanding data.
+
+### 2.2 Know your users
+This is the obvious start of most Information retrieval projects but, until now, I haven’t seen it frequently mentioned in a RAG context: know what your users are doing with your systems.
+
+Overall, RAG applications underestimate the continuity with the previous expert system and the inertia of user practices. I recently analyzed a set of internal user queries with a chatbot in production. One of the most surprising results is that many users were not submitting questions and conversations but unstructured keywords. This is perfectly fitting in the standard information retrieval system, but instruction-based LLM and embedding models are by design trained on question/answer pairs.
+
+Dissociation between use and design should be a primary concern. Without proper care, LLMs and embedding models are not only unable to solve existing data frictions but will introduce new frictions : alignment mismatch with user expectation, poor breakdown of data structure due to tokenization, incongruous length, total lack of control over the system purpose (the infamous “Chevrolet” effect).
+
+### 2.3 Know your data
+Like your users, the data will not necessarily agree with the constraints of language models and embedding models.
+
+Context window is an obvious restriction. Despite its gradual lengthening, the indexation of long texts always underperforms. This is one of the biggest mismatches between pre-training data and data in production: the vast majority of Common Crawl texts are less than 512 tokens. Organizations routinely deal with multi-pages PDF.
+
+So you need to split or chunk your document and… this is not trivial. Documents parts are not meant to be standalone units of content and disambiguation requires some level of cross-referencing. The biggest performance gain on my retrieval evaluation was the… systematic inclusion of title hierarchy (document > chapter > section etc.). Simply because in long documents, general information on the topic is given at this scale, but not recalled in specific paragraphs.
+
```

Comment:
"my retrieval evaluation" - you're referring to a specific eval you carried out, or several, or... what is the reference?